internal/pkg/log/writer.go

```diff
@@ -33,17 +33,17 @@ func (w *LevelWriter) write(p []byte) (n int, err error) {
 	return len(p), nil
 }
 
-func (w *LevelWriter) Write(p []byte) {
-	if _, err := w.write(p); err != nil {
+func (w *LevelWriter) WriteNoErr(p []byte) {
+	if _, err := w.Write(p); err != nil {
 		panic(fmt.Errorf("cannot write: %w", err))
 	}
 }
 
 func (w *LevelWriter) WriteString(s string) {
-	w.Write([]byte(s))
+	w.WriteNoErr([]byte(s))
 }
 
-func (w *LevelWriter) WriteStringIndent(s string, indent int) {
+func (w *LevelWriter) WriteStringIndent(indent int, s string) {
```

Comment:
Vymenene poradie parametrov, kedze `indent` je vzdy kratke cislo, .... a takto sa to lepsie cita.